applications/nrf_desktop/src/modules/dev_descr.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <zephyr/types.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/gatt.h>
+
+#include "dev_descr.h"
+
+static u8_t device_descr[] = {
```

Comment:
const?

Reply:
This would cause compiler warning when defining services and characteristics for GATT.